comp/core/tagger/taggerimpl/collectors/workloadmeta_extract.go

```diff
@@ -49,6 +49,10 @@ const (
 	envVarVersion = "DD_VERSION"
 	envVarService = "DD_SERVICE"
 
+	// OpenTelemetry SDK - Environment variables
```

Comment:
We should add a link to the OpenTelemetry docs here. For future reference, to know where these 2 ENVs are defined.

Reply:
Done 👍